backend/API/API/Services/MessageService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Domains.Models.Input;
+using Domains.Models.Output;
+using Microsoft.Extensions.Configuration;
+using QueueService.Interfaces;
+using QueueService.Models;
+
+namespace API.Services
+{
+    public class MessageService
+    {
+        IConnectionProvider factory;
+        IConfiguration configuration;
+
+        public MessageService(IConnectionProvider factory, IConfiguration configuration)
+        {
+            this.factory = factory;
+            this.configuration = configuration;
+        }
+
+        public void SendFileData(FileInput fileInput)
+        {
+            Settings settings = new Settings();
+            configuration.Bind("RabbitMq:FileData", settings);
+            using (IProducer producer = factory.Open(settings))
+            {
+                producer.Send(fileInput);
+            }
+        }
+
+        public FileOutput DequeueData()
+        {
+            Settings settings = new Settings();
+            configuration.Bind("RabbitMq:Result", settings);
+            using (IConsumer consumer = factory.Connect(settings))
+            {
+                ReceiveData receiveData = consumer.Receive(500);
+                if (receiveData == null) return null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4618369)